eclair-core/src/test/scala/fr/acinq/eclair/router/GraphSpec.scala

```diff
@@ -348,4 +348,52 @@ class GraphSpec extends AnyFunSuite {
     assert(paths(1).path == Seq(edgeCE, edgeEG, edgeGH))
     assert(paths(2).path == Seq(edgeCD, edgeDF, edgeFH))
   }
+
+  test("RoutingHeuristics.normalize when value in range") {
+    assert(Graph.RoutingHeuristics.normalize(value = 10, min = 0, max = 100) == 0.100008)
+    assert(Graph.RoutingHeuristics.normalize(value = 20, min = 10, max = 200) == 0.05264052631578948)
+    assert(Graph.RoutingHeuristics.normalize(value = -11, min = -100, max = -10) == 0.9888791111111112)
+  }
```

Comment:
This test only tells us that the code hasn't changed but it doesn't test any property we care about. We don't want `normalize(10, 0, 100) == 0.100008`, we want it to be `0.1` and we tolerate some small deviation.

Reply:
I changed to the other proposed implementation so that now we get 0.1 for this case.

Reply:
OK - If you prefer the original implementation, I can switch back to it and add a tolerance to the test.